src/components/Atoms/AddTaskButton/index.jsx

```diff
@@ -0,0 +1,40 @@
+import plus from "../../../assets/svg/plus.svg";
+import styled from "styled-components";
+import React from "react";
+import COLOR from "../../../variables/color";
+import TEXTS from "../../../variables/texts";
+import FONT_FAMILY from "../../../variables/font_family";
+
+export const AddTaskButton = ({ onClick }) => {
+  return (
+    <StyledAddTaskButton onClick={onClick}>
+      <img src={plus} className="PlusButton" />
+      <div className="AddTask">タスクを追加</div>
```

Comment:
案件で言われることなのですが、
Reactではなるべくdivタグ等の一般的なタグやclassNameを使わず、
自分で変数名を設定するようにしましょう。
※imgタグはそのままでもいいですが、今回のようにWrapper等を使わずに
　ダイレクトに装飾を加える場合、新しい変数を追加した方がよいかと思います

```suggestion
      <StyledImg src={plus} />
      <StyledButton>タスクを追加</StyledButton>
```